agroal-api/src/main/java/io/agroal/api/configuration/AgroalConnectionPoolConfiguration.java

```diff
@@ -122,6 +122,7 @@
     /**
      * The maximum amount of time a thread may be blocked waiting for a connection. If this time expires and still no connection is available, an exception is thrown.
      * A duration of {@link Duration#ZERO} means that a thread will wait indefinitely.
+     * In Pool-less this timeout can add to {@link AgroalConnectionFactoryConfiguration#loginTimeout()}.
```

Comment:
In Pool-less, there is no background thread and therefore there can be a login timeout period after an (almost) acquisition timeout period. Added a note here and will update documentation as well.